client/src/components/RestaurantList.js

```diff
@@ -31,11 +63,13 @@ const RestaurantList = () => {
         )
     })
 
-    return(
+    return (
         <div>
             <h1>Radical Reviews</h1>
             <ul>{restaurantList}</ul>
-        </div>   
+            <ErrorList errors={errors} />
+            <NewRestaurantForm addRestaurant={addRestaurant} />
+        </div>
```

Comment:
this is okay for now, but this type of form may work better if it exists on its own page. Again, fine for now, but something you may consider changing up in the future